test/test_torch.py

```diff
@@ -6088,6 +6088,101 @@ def test_pow(self, device):
             torch.pow(m1, 1, out=out)
             self.assertEqual(out, m1)
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
```

Comment:
We now require NumPy to run the test suite and these checks are no longer necessary.

Reply:
Addressed!